pkg/kubectl/cmd/create_clusterrole.go

```diff
@@ -133,6 +134,12 @@ func (c *CreateClusterRoleOptions) Validate() error {
 				return fmt.Errorf("invalid verb: '%s' for nonResourceURL", v)
 			}
 		}
+
+		for _, nonResourceURL := range c.NonResourceURLs {
+			if nonResourceURL != "*" && !strings.HasPrefix(nonResourceURL, "/") {
+				return fmt.Errorf("invalid nonResourceURL: '%s'", nonResourceURL)
```

Comment:
make the message a little more helpful (like "nonResourceURL should start with /")